cmd/cql/internal/create.go

```diff
@@ -0,0 +1,72 @@
+package internal
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/CovenantSQL/CovenantSQL/client"
+)
+
+var CmdCreate = &Command{
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var CmdCreate should have comment or be unexported